karma.conf.js

```diff
@@ -7,6 +7,9 @@ module.exports = function (config) {
         singleRun: true,
         files: [],
         logLevel: config.LOG_ERROR,
+        browserConsoleLogOptions: {
+            level: 'warn'
+        },
```

Comment:
reducing output of tests to warnings only - there was too much trash being logged to the terminal